[quant][graphmode][fix] Fix insert quant dequant for observers without qparams

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#43606 [quant][graphmode][fix] Fix insert quant dequant for observers without qparams**

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23335106](https://our.internmc.facebook.com/intern/diff/D23335106)

jerryzh168 committed:
[quant][graphmode][fix] Fix insert quant dequant for observers without qparams

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar commented on torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp:
```diff
@@ -392,6 +392,11 @@ void insertQuantizationOps(
     }
     return;
   }
+  if (qparam_names.size() == 0) {
```
Isn't this covered by the case for EmbeddingQuant above on L391?

supriyar commented:
This fix isn't needed, there is an issue with the test. Clarified more on the chat thread.

jerryzh168 committed:
Update on "[quant][graphmode][fix] Fix insert quant dequant for observers without qparams"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23335106](https://our.internmc.facebook.com/intern/diff/D23335106)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Fix insert quant dequant for observers without qparams"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23335106](https://our.internmc.facebook.com/intern/diff/D23335106)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Fix insert quant dequant for observers without qparams"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23335106](https://our.internmc.facebook.com/intern/diff/D23335106)

[ghstack-poisoned]
